resources/views/registerProject.blade.php

```diff
@@ -0,0 +1,218 @@
+@extends('layouts.app')
+
+@section('title', 'Add Project')
+
+@section('content')
+<!-- Internal CSS stylesheet -->
+<style>
+	#uploadImage{
+		margin-top: 1em;
+		margin-bottom: 2em;
+	}
+
+	.hidden{
+		display: none;
+	}
+</style>
+<div class="ui container" style="margin: 4% auto 8%">
+	<div class="ui grid centered">
+		<div class="ten wide column">
+			<div class="ui form error">
+				<!-- Alert Message -->
+				<div id="errorMessage" class="ui error message hidden">
+					<div class="header">Incomplete Information</div>
+					<p>All fields are required for registering a new project.</p>
+				</div>
+				<!-- Project Image -->
+				<div class="projectImg">
+					<img src="https://lorempixel.com/400/400" alt="Project Image" class="ui centered medium circular image"/>
+					<a href="#" class="imgUploader">
+						<button id="uploadImage" class="fluid ui button">Upload Image</button>
+					</a>
+					<input type="file" id="imgInput" style="display:none" multiple accept='image/*'>
+				</div>
+				<!-- Project Name -->
+				<div id ="projectName" class="field">
+					<label for="proyectName">Project Name</label>
+					<input type="text" name="projectName" placeholder="Enter your project's name...">
+				</div>
+				<!-- Team Name -->
+				<div id="teamName" class="field">
+					<label for="teamName">Team Name</label>
+					<input type="text" name="teamName" placeholder="Enter your team's name...">
+				</div>
+				<!-- Category -->
+					<!-- <div id="category" class="field">
+						<label>Category</label>
+						<div class="ui fluid search selection dropdown category">
+							<i class="dropdown icon"></i>
+							<input id ="category" type="hidden" name="country">
+							<div class="default text">Select a category of your project</div>
+							<div class="menu">
+								<div class="item" value="1">Category 1</div>
+								<div class="item" value="2">Category 2</div>
+								<div class="item" value="3">Category 3</div>
+								<div class="item" value="4">Category 4</div>
+							</div>
+						</div>
+					</div> -->
+				<div id ="category" class="field">
+					<label for="category">Category</label>
+					<select name="category" class="ui search dropdown" placeholder="Select your project's category">
```

Comment:
The dropdown doesn't seem consistent with the rest of the inputs. When nothing is selected, the color should be grayed out with the **same color as the placeholders of the other inputs.**


![image](https://user-images.githubusercontent.com/14140121/53279473-57799d80-36d6-11e9-805e-a6fa500cc8f3.png)